openedx/core/djangoapps/content_libraries/tests/test_content_libraries.py

```diff
@@ -289,12 +240,12 @@ def test_library_blocks(self):
         assert lib['has_unpublished_changes'] is False
 
         # A library starts out empty:
-        assert self._get_library_blocks(lib_id) == []
+        assert self._get_library_blocks(lib_id)['results'] == []
 
         # Add a 'problem' XBlock to the library:
         block_data = self._add_block_to_library(lib_id, "problem", "problem1")
         self.assertDictContainsEntries(block_data, {
-            "id": "lb:CL-TEST:testlib1:problem:problem1",
+            "id": "lb:CL-TعST:testlꜟط:problem:ࠒröblæm1",  # Non-ascii slugs to test unicode robustness.
```

Comment:
I feel like I'm missing something here... this is checking that the _output_ contains these unicode characters, but the _input_ just a few lines above is ASCII - `problem1` on line 246 for the block_id and `testlib1` on the first line of the function for the library ID. Where are the unicode versions originally coming from?

Reply:
@bradenmacdonald the only thing you're missing is that I made the wrong change :) Locally, this test is failing the way you'd expect it to--it hasn't shown up in CI yet because another test failure is cancelling its shard.